src/NHibernate/Util/ReflectHelper.cs

```diff
@@ -407,35 +407,34 @@ public static System.Type TypeFromAssembly(AssemblyQualifiedTypeName name, bool
 				if (name.Assembly == null)
 				{
 					// No assembly was specified for the type, so just fail
-					string message = "Could not load type " + name + ". Possible cause: no assembly name specified.";
-					log.Warn(message);
-					if (throwOnError) throw new TypeLoadException(message);
+					log.Warn("Could not load type {0}. Possible cause: no assembly name specified.", name);
+					if (throwOnError) throw new TypeLoadException("Could not load type " + name + ". Possible cause: no assembly name specified.");
```

Comment:
Here previous code was better in my opinion. Now the message may be constructed two times by two different means.

Reply:
Still. The message in exception needs to be updated to use string.Format